allowed_libraries.conf

```
@@ -15,6 +15,10 @@ libsailfishapp.so.1
 libmdeclarativecache5.so.0
 libsailfishsilica.so.1
 
+# ### Sailfish Secrets framework
+libsailfishcrypto.so.0
```

```suggestion
libsailfishsecretscrypto.so
```
Afais this should like this.

---

Why? The library is libsailfishcrypto.so.0, there is no secrets in the name

---

It's actually both...
```
rpm -ql libsailfishsecretscrypto
/usr/lib64/libsailfishsecretscrypto.so.0
/usr/lib64/libsailfishsecretscrypto.so.0.2
/usr/lib64/libsailfishsecretscrypto.so.0.2.32
```

---

Added libsailfishsecretscrypto. Also added the QML APIs and the Secrets permission.